src/_sass/layout.scss

```
@@ -1423,3 +1423,23 @@ a.attached-above {
   font-weight: bold;
   margin-top: - $spacing-unit / 2
 }
+
+.pagination {
+  display: flex;
+  margin: - $spacing-unit auto $spacing-unit auto;
```

When the pagination appears at the bottom of the page, it's a little too close to the blog excerpts and can get visually lost. If we leave more room then people are less likely to miss this additional content.

```suggestion
  margin: $spacing-unit auto $spacing-unit auto;
```

---

agreed, i added more spacing to the bottom version